test/pytest/test_bridge.py

```diff
@@ -474,6 +474,30 @@ async def test_fslist1_notexist(transport: MockTransport) -> None:
         reply_keys={'message': "[Errno 2] No such file or directory: '/nonexisting'"})
 
 
+@pytest.mark.asyncio
+async def test_fsreplace1(transport: MockTransport, tmp_path: Path) -> None:
+    # create non-existing file
+    myfile = tmp_path / 'newfile'
+    ch = await transport.check_open('fsreplace1', path=str(myfile))
+    transport.send_data(ch, b'some stuff')
+    transport.send_done(ch)
+    await transport.assert_msg('', command='done', channel=ch)
+    await transport.check_close(channel=ch)
+    assert myfile.read_text() == 'some stuff'
+    # no leftover files
+    assert os.listdir(tmp_path) == ['newfile']
+
+    # now update its contents
```

Comment:
Please add these two cases:
 - sending one empty data message to confirm that an empty file is created
 - sending no data message to confirm that the file is deleted